### ISSUE TITLE
Label(Database throughput) is not associated with radio controls 'Auto scale' and 'Manual' of 'New collection' blade.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1028]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1217594

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 
Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Add Collection" Menu item/Button and select it.
Navigate through all the controls under add Collection/New collection blade.
Navigate to radio controls 'Auto scale' and 'Manual' of 'New collection' blade.
Press F12 key to open the Developers Tool and inspect the code.
Verify the issue.

Actual Result:
Label(Database throughput) is not associated with radio controls 'Auto scale' and 'Manual' of 'New collection' blade due which screen reader is not reading label information.

Expected Result:
Label(Database throughput) should be associated with radio controls 'Auto scale' and 'Manual' of 'New collection' blade so that the screen reader will provide label information and define the purpose of radio controls.

Note:
Similar issue exits for edit field "Database Max RU/s". Refer to the screenshot: [MAS3.3.2-Label not associated with edit field.]
This issue also exists under 'New Database' blade for similar controls.
This issue also exists for similar controls present under 'New Graph' blade of Data explorer screen for Created cosmos DB  after selecting "Gremlin(graph)" option.
This issue exists throughout the application for similar controls like edit field and radio controls.


![image](https://user-images.githubusercontent.com/8519407/132720692-ef6980e6-1f34-4607-b15b-40b2aa8dae0e.png)

![image](https://user-images.githubusercontent.com/8519407/132720715-4ed63a68-4b2b-47a1-8451-b14be75e994e.png)
